src/main/src/main/java/org/geoserver/config/ServiceInfo.java

```diff
@@ -238,4 +238,10 @@
     
     Map<Object,Object> getClientProperties();
 
+    /**
+     * 
```

Comment:
Lacks Javadoc in an interface. The "allowEnvParametrization" seems more like a "replaceParameters" to me